tools/ds-identify

```diff
@@ -1384,16 +1383,20 @@ vmware_has_rpctool() {
     command -v vmware-rpctool >/dev/null 2>&1
 }
 
+vmware_rpctool_guestinfo() {
+    vmware-rpctool "info-get guestinfo.${1}" 2>/dev/null | grep "[[:alnum:]]"
+}
+
 vmware_rpctool_guestinfo_metadata() {
-    vmware-rpctool "info-get guestinfo.metadata"
+    vmware_rpctool_guestinfo "metadata"
 }
 
 vmware_rpctool_guestinfo_userdata() {
-    vmware-rpctool "info-get guestinfo.userdata"
+    vmware_rpctool_guestinfo "userdata"
 }
 
 vmware_rpctool_guestinfo_vendordata() {
-    vmware-rpctool "info-get guestinfo.vendordata"
+     vmware_rpctool_guestinfo "vendordata"
```

Comment:
You copied my original example verbatim and missed the thing I missed too. There's a leading WS on line 1399 :)

Reply:
Sigh. It's 3am :)

Reply:
Right, so go to bed! :) I was wondering why you were poking at this in the first place so late. We're not spring chickens anymore Shreenidhi :)